siocli/lib/SqlDump.cpp

```diff
@@ -0,0 +1,771 @@
+    // ss << ", " << kSysDatabases_CipherKey_ColumnName << " = " << dbInfo.m_cipherKey;
+
+    return ss.str();
+}
+
+std::string formCreateTableQuery(const std::string& databaseName, const TableInfo& tableInfo)
+{
+    std::stringstream ss;
+
+    ss << "CREATE TABLE " << databaseName << '.' << tableInfo.m_tableName;
+
+    if (!tableInfo.m_columns.empty()) {
+        ss << " (";
+
+        for (auto i = 0u; i < tableInfo.m_columns.size() - 1; ++i) {
+            const auto& column = tableInfo.m_columns[i];
+            ss << " " << column.m_name << " " << getColumnDataTypeName(column.m_dataType);
+            for (const auto& constraint : column.m_constraints)
+                ss << " " << constraintToString(constraint);
+
+            ss << ',';
+        }
+
+        const auto& column = tableInfo.m_columns.back();
+        ss << " " << column.m_name << " " << getColumnDataTypeName(column.m_dataType);
+        for (const auto& constraint : column.m_constraints)
+            ss << " " << constraintToString(constraint);
+
+        ss << " )";
+    }
+
+    return ss.str();
+}
+
+BinaryValue readBinaryValue(google::protobuf::io::CodedInputStream& input)
+{
+    std::uint32_t blobLength = 0;
+    // Read length
+    if (!input.ReadVarint32(&blobLength))
+        throw std::runtime_error("Can't read binary value length");
+
+    BinaryValue value;
+    value.resize(blobLength);
+
+    if (!input.ReadRaw(value.data(), blobLength))
+        throw std::runtime_error("Binary data read failed");
+
+    return value;
+}
+
+std::string readString(google::protobuf::io::CodedInputStream& input)
+{
+    std::uint32_t clobLength = 0;
+    // Read length
+    if (!input.ReadVarint32(&clobLength))
+        throw std::runtime_error("Can't read string value length");
+
+    // Read sample
+    std::string str;
+    str.resize(clobLength);
+
+    if (!input.ReadRaw(str.data(), clobLength)) throw std::runtime_error("String data read failed");
+
+    return str;
+}
+
+std::int8_t readInt8(google::protobuf::io::CodedInputStream& input)
+{
+    std::int8_t data = 0;
+    if (!input.ReadRaw(&data, sizeof(data))) throw std::runtime_error("Read int8 failed");
+
+    return data;
+}
+
+std::int8_t readInt16(google::protobuf::io::CodedInputStream& input)
+{
+    std::int16_t data = 0;
+    if (!input.ReadRaw(&data, sizeof(data))) throw std::runtime_error("Read int16 failed");
+
+    return data;
+}
+
+std::int32_t readInt32(google::protobuf::io::CodedInputStream& input)
+{
+    std::int32_t data = 0;
+    if (!input.ReadVarint32(reinterpret_cast<std::uint32_t*>(&data)))
+        throw std::runtime_error("Read int32 failed");
+
+    return data;
+}
+
+std::int64_t readInt64(google::protobuf::io::CodedInputStream& input)
+{
+    std::int64_t data = 0;
+    if (!input.ReadVarint64(reinterpret_cast<std::uint64_t*>(&data)))
+        throw std::runtime_error("Read int64 failed");
+
+    return data;
+}
+
+std::uint8_t readUint8(google::protobuf::io::CodedInputStream& input)
```

Comment:
please follow existing naming convention which is `UInt`, not "Uint"
Fix everywhere